hack/convert-output-to-tsv.py

```diff
@@ -0,0 +1,42 @@
+# A script to convert transformed data into TSV files
+# Note: Before uploading to Terra, manually go in and update PK column name to be prefixed by "entity:"
```

Comment:
I did this manually since I didn't want to mess with all of the JSON dictionaries. I'd be happy to add it to the script though if anyone prefers that.

Reply:
If it isn't a huge headache, I think it'd be worth adding